tools/breaking-changes/generate-methods-props.ts

```diff
@@ -105,13 +107,21 @@ function isMethodOrProperty(memberKind: string): boolean {
   return memberKind.startsWith('Method') || memberKind.startsWith('Property');
 }
 
-function normalizeDocSignature(doc: string): string {
-  return normalizeNewLines(doc).replace(/\s+/g, ' ').trim();
+function normalizeFunctionSignatureDoc(doc: string): string {
+  return normalizeNewLinesForFunction(doc).replace(/\s+/g, ' ').trim();
 }
 
-function normalizeNewLines(doc: string) {
+function normalizeNewLinesForFunction(doc: string) {
   doc = doc.slice(1).slice(0, -1); // Remove first and last chars.  They are \n.
   doc = doc.replace(/\(\n/, '('); // Remove \n next to the opening parenthesis.
   doc = doc.replace(/\n\)/, ' )'); // Remove \n next to the closing parenthesis.
   return doc.replace(/\n/g, ','); // The rest of the \n are between params.  Replace with `,`
 }
+
+function normalizePropertyDoc(doc: string): string {
+  return normalizeNewLinesForProperty(doc).replace(/\s+/g, ' ').trim();
```

Comment:
What do you think about

doc = doc.replace(/\n/g, '');
doc = doc.replace(/\s+/g, ' ').trim();

return doc;